Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,14 +1,65 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.IO;
+using System.Globalization;
 using Console = System.Console;
 
 namespace BootCamp.Chapter
 {
     class Program
     {
+        static string firstName;
```

Comment:
Declare all those as close as possible to where they are used. Declaring all at the top is not so readable.